### DISCUSSION TITLE
How to update prisma client's types if schema was updated?

### DISCUSSION THREAD

**Discussion by SuperManEver** [prisma/prisma#4568] [Q&A] [answered]
I've updated a `schema.prisma` by adding a new field to one of the models but Prisma client complain about that

`Object literal may only specify known properties, and 'rating' does not exist in type 'PostCreateInput'.`

I guess that I should somehow update Prisma client's types in order to reflect the update I made to the schema.

How should I do that?

**Comment by ryands17** [comment, marked as answer]
Hey @SuperManEver 👋 

You need to run `prisma generate` to update the types to reflect the schema changes. You can read more about that [here](https://www.prisma.io/docs/concepts/components/prisma-schema).

**Reply by billymoonz** [reply]
doesn't even work...

**Reply by janpio** [reply]
Can you be more concrete what is not working?

Most probably you need to run `npx prisma generate` in your project folder to execute that CLI command.

**Reply by ceconcarlsen** [reply]
> doesn't even work...

u can try `npx prisma db pull`

**Reply by chaboxx** [reply]
Thanks boys

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
